qa-data/invalid-data-delimiter/data.csv

```diff
@@ -0,0 +1,3 @@
+id,city,,,,,
```

Comment:
@Mikanebu I think it is called inconsistent number of columns (extra delimiters), but not invalid delimiter.